### DISCUSSION TITLE
找不到lamp-parnet依赖

### DISCUSSION THREAD

**Discussion by nizpxox** [zuihou/zuihou-admin-cloud#307] [01.常见问题]
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
 @ 
[ERROR] The build could not read 12 projects -> [Help 1]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-dependencies-parent:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-public:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-public\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-base:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-base\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-oauth:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-oauth\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-system:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-system\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-gateway:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-gateway\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-support:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-support\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-generator:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-generator\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-activiti:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-activiti\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-child:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-child\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-production:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-production\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-framework:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-framework\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in http://maven.aliyun.com/nexus/content/groups/public/ during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException

**Comment by nizpxox** [comment]
是镜像源不行嘛

**Reply by nizpxox** [reply]
换了一个镜像还是不行

**Reply by nizpxox** [reply]
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in https://maven.aliyun.com/repository/public during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project top.tangyh.lamp:lamp-framework-api:4.14.0 (D:\Software\lamp-cloud-pro-datasource-column\lamp-framework\lamp-framework-api\pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for top.tangyh.lamp:lamp-dependencies-parent:4.14.0: The following artifacts could not be resolved: top.tangyh.basic:lamp-parent:pom:4.14.0 (absent): top.tangyh.basic:lamp-parent:pom:4.14.0 was not found in https://maven.aliyun.com/repository/public during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of alimaven has elapsed or updates are forced and 'parent.relativePath' points at no local POM @ top.tangyh.lamp:lamp-dependencies-parent:4.14.0, D:\Software\lamp-cloud-pro-datasource-column\lamp-dependencies-parent\pom.xml, line 6, column 13 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:

**Comment by zuihou** [comment]
需要自行下载lamp-util-pro 项目的源码，在自己本地编译
